acm/src/main/java/org/literacybridge/acm/gui/settings/AcmGeneralSettingsPanel.java

```diff
@@ -54,6 +61,7 @@ public String getTitle() {
 
         // Get values from the global configuration.
         threshold = ACMConfiguration.getInstance().getCurrentDB().getFuzzyThreshold();
+        intVolumeStep = ACMConfiguration.getInstance().getCurrentDB().getVolumeStep();
```

Comment:
When you fix getVolumeStep() you'll need to properly handle when intVolumeStep is null. Also, it is not an "int", so don't call it that, just "volumeStep". Furthermore, we don't use Hungarian notation.

Reply:
Checks for whether volumeStep is null or not is handled at line 148. It sets the edit text field to 0 if volume step value is null, otherwise it uses the value.